src/runner/tests/driver-test-runner.js

```diff
@@ -0,0 +1,11 @@
+import test from 'ava';
+
+import { isDesiredSettingResponse } from '../driver-test-runner.js';
+
+test('isDesiredSettingResponse', t => {
+  t.true(isDesiredSettingResponse('BROwse ModE', ['Browse mode']), 'handles casing');
+
+  t.true(isDesiredSettingResponse('quick nav on', ['Quick nav on', 'All quick nav on']));
+
+  t.false(isDesiredSettingResponse('quick nav', ['Quick nav on', 'All quick nav on']));
+});
```

Comment:
@jugglinmike how's this for a simplified set of unit tests here?

Reply:
Way more clear!